data/web/templates/edit/alias.twig

```diff
@@ -11,14 +11,37 @@
   {% endif %}
   <div class="row mb-2">
     <label class="control-label col-sm-2" for="address">{{ lang.edit.alias }}</label>
+  {% if result.is_wildcard == '1' %}
+    <div class="col-sm-10 d-flex justify-content-start align-items-center mb-2">
+        <div class="col-sm-5 col-md-5">
+            <input type="text" pattern="[A-Za-z0-9\.!#$%&'*+/=?^_`{|}~-]+" autocorrect="off" autocapitalize="none" class="form-control" name="local_part" value="{{ result.address|split('@')[0] }}" required>
```

Comment:
What does the pattern validates?
It seems like the form is to edit the alias and the regex seems to allow characters that should not be allowed in email address.
Although I'm not sure about the usage context of this form

Reply:
Yes, you understand the context,
But this is like the minimized JS, It's to keep code consistency. I don't know why they used this pattern for local field 🤷
Comming from : 0b64967ec56f14bbdb288e07b02d478c835be513

Reply:
If it's for consistency, I agree with you in that case.
Although I didn't find the regex in the reference commit you linked

Reply:
Originally from a commit 7 years ago agod891bc88945e2be5b2af4a3db6f9ec5457705b7b
There's no explanation, maybe it's an old pattern issue, but I'm not sure.
I can update it, but how testing side effect for this ?

You can't find it `406 changed files with 34,644 additions and 7,408 deletions.`, the only way is to use a lot of git command lines :(

Reply:
haha I see
you are right, to avoid side effect, let's not change it

Reply:
you can resolve